.github/policies/labelAdded.validationUnattendedFailed.yml

```diff
@@ -28,7 +28,7 @@ configuration:
                 During installation testing, this application failed to install without user input. Did you forget to add Silent or SilentWithProgress switches?
 
 
-                This can also happen when a dependency is missing. You can test with https://github.com/microsoft/winget-pkgs#test-your-manifest.
+                This can also happen when a dependency is missing. You can test based on the [documentation](https://github.com/microsoft/winget-pkgs/blob/HADE/doc/README.md#testing-a-manifest).
```

Comment:
```suggestion
                This can also happen when a dependency is missing. You can test based on the [documentation](https://github.com/microsoft/winget-pkgs/blob/master/doc/README.md#testing-a-manifest).
```